### PR TITLE
Update full-stack image version - aiida v2.6

### PR DESCRIPTION
**Pull request by danielhollas**
I just released a new version of the AiidaLab docker image with AiiDA 2.6.2. This updates QE image on top of that.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -8,12 +8,17 @@ build-backend = "setuptools.build_meta"
...
-    'ignore:Object of type .* not in session, .* operation along .* will not proceed:',
+    # This is needed since SQLAlchemy 2.0, see
+    # https://github.com/aiidalab/aiidalab-widgets-base/issues/605
+    'ignore:Object of type.*not in session,.*operation along.*will not proceed:',
     'ignore:Creating AiiDA configuration:UserWarning:',
     'ignore:metadata.*traitlets.traitlets.Unicode object:DeprecationWarning:',
     # For some reason we get this error, see
     # https://github.com/aiidalab/aiidalab-widgets-base/issues/551
     'ignore:Exception ignored in:pytest.PytestUnraisableExceptionWarning:_pytest',
+    "ignore::cryptography.utils.CryptographyDeprecationWarning:",
```
Comment (danielhollas) [inline pyproject.toml:19]:
```suggestion
```

**Comment by danielhollas** [comment]
@unkcpz I believe this one should be merged before #795 since it fixes one tests for aiida v2.6

**Comment by unkcpz** [comment]
@danielhollas there are two changes of docker stack merged and used by #795 but not included in the release. Could you make a release of it?

**Comment by danielhollas** [comment]
@unkcpz feel free to make a release!

**Pull request merged**